charts/dev/livebook/values.yaml

```diff
@@ -19,12 +19,28 @@ service:
     ports:
       main:
         port: 8787
-        protocol: tcp
-        targetPort: 8080
+  headless:
+    enabled: true
+    primary: false
+    clusterIP: None
+    ports:
+      headless:
+        enabled: true
+        port: "{{ .Values.service.main.ports.main.port }}"
 workload:
   main:
     podSpec:
       containers:
         main:
           env:
-            LIVEBOOK_PASSWORD: livebookiscool
+            # A_ because these variables are sorted alphabetically and this one needs to come first.
+            A_POD_IP:
+              fieldRef:
+                fieldPath: status.podIP
+            LIVEBOOK_CLUSTER: '{{ printf "dns:%s-headless" (include "tc.v1.common.lib.chart.names.fullname" .) }}'
+            LIVEBOOK_DISTRIBUTION: name
+            LIVEBOOK_NODE: livebook@$(A_POD_IP)
+            LIVEBOOK_PORT: "{{ .Values.service.main.ports.main.port }}"
+            LIVEBOOK_HOME: "{{ .Values.persistence.data.mountPath }}"
+            LIVEBOOK_SECRET_KEY_BASE: "{{ randAlphaNum 48 | b64enc }}"
```

Comment:
I'd say make this a "remembered secret" (see firefly iii)
I assume this is used to store user sessions, Currently on every restart it will kick people out and have to relogin

Reply:
Oh I see what you mean. True that. Okay I ckeck in firefly iii if I can find out how to do this.

Reply:
templates/_secrets.tpl => That what you referring to?

Reply:
Yep!

Reply:
gotcha, thanks.

Reply:
Same for LIVEBOOK_COOKIE?

Reply:
added that to the secret, too.